git_machete/cli.py

```diff
@@ -0,0 +1,540 @@
+#!/usr/bin/env python
+
+import getopt
+import os
+import re
+import sys
+import textwrap
+from typing import Dict, List, Optional, Tuple, TypeVar
+
+from git_machete import __version__
+from git_machete import utils
+from git_machete.client import MacheteClient, alias_by_command, allowed_directions
+from git_machete.constants import EscapeCodes
+from git_machete.docs import short_docs, long_docs
+from git_machete.exceptions import MacheteException, StopTraversal
+from git_machete.git_operations import GitContext
+from git_machete.options import CommandLineOptions
+from git_machete.utils import fmt, underline, excluding, warn
+
+T = TypeVar('T')
+
+initial_current_directory: Optional[str] = utils.get_current_directory_or_none() or os.getenv('PWD')
+
+command_by_alias: Dict[str, str] = {v: k for k, v in alias_by_command.items()}
+
+command_groups: List[Tuple[str, List[str]]] = [
+    # 'is-managed' is mostly for scripting use and therefore skipped
+    ("General topics", ["file", "help", "hooks", "version"]),
+    ("Build, display and modify the tree of branch dependencies",
```

Comment:
Nit: I'd merge this line (and 33/34) for clarity

Reply:
Instead I split this in such way that there is:
(str
 List),
(str
 List),
...
So each pair takes 2 lines in the same format and order.